usaspending_api/awards/serializers.py

```diff
@@ -22,11 +22,27 @@ class Meta:
         fields = '__all__'
 
 
+class ProcurementSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Procurement
+        fields = '__all__'
+
+
+class FinancialAssistanceAwardSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = FinancialAssistanceAward
+        fields = '__all__'
+
+
 class AwardSerializer(serializers.ModelSerializer):
 
     recipient = LegalEntitySerializer(read_only=True)
     awarding_agency = AgencySerializer(read_only=True)
     funding_agency = AgencySerializer(read_only=True)
+    procurement_set = ProcurementSerializer(many=True, read_only=True)
```

Comment:
should there be logic in the serializer that only shows procurement if it's a procurement award and vice versa for financial assistance? Or if these are null will the key not show up?